.gitignore

```diff
@@ -170,4 +170,3 @@ app.*.symbols
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
 !/dev/ci/**/Gemfile.lock
 !.vscode/settings.json
-!**/pubspec.lock
```

Comment:
Just a side note: I added this back in originally (an exception to the generated files rule) to comply with application development. More so if someone runs this later (another semester perhaps) and there are dependency issues, the application will still run:
- https://stackoverflow.com/questions/16136739/in-dart-and-pub-should-i-add-pubspec-lock-to-my-gitignore
- https://medium.com/codingmountain-blog/what-is-pubspec-lock-and-why-should-flutter-devs-care-86ab4d0b47fc